open-craft/edx-analytics-pipeline
# fix: adds pbr package to the python requirements

## Description

This change resolves the following issue which was showing up in the edx analytics tasks for edx-olive.

```
python setup.py clean
Couldn't find index page for 'pbr' (maybe misspelled?)
No local packages or working download links found for pbr
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    pbr=True
  File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
    _setup_distribution = dist = klass(attrs)
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/setuptools/dist.py", line 335, in __init__
    self.fetch_build_eggs(attrs['setup_requires'])
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/setuptools/dist.py", line 456, in fetch_build_eggs
    replace_conflicting=True,
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 863, in resolve
    replace_conflicting=replace_conflicting
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1141, in best_match
    return self.obtain(req, installer)
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1153, in obtain
    return installer(requirement)
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/setuptools/dist.py", line 522, in fetch_build_egg
    return cmd.easy_install(req)
  File "/edx/var/jenkins/workspace/course-activity-workflow/venvs/analytics-tasks/local/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 666, in easy_install
    raise DistutilsError(msg)
distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pbr')
Makefile:5: recipe for target 'uninstall' failed
```

## Supporting information

* **JIRA Tickets**: SE-4221

## Testing instructions

* Build a new job using this branch instead, and verify that the same issue doesn't happen (approximately 14 mins in).

## Deadline

ASAP 

## Other information

This change has already been tested by creating a new build with the change from this pull request. It worked as expected. So I'll be merging it asap to resolve the other builds.
